types/fast-fifo/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6"],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "baseUrl": "../",
+        "typeRoots": ["../"],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

Comment:
Based on the exports, it looks like you want to remote `esModuleInterop` and use `import FIFO = require('fast-fifo');` in the tests.

Reply:
I've made this change.